proposals/0304-structured-concurrency.md

```diff
@@ -609,17 +606,17 @@ A new, detached task can be created with the `detach` operation. The resulting t
 ```swift
 /// Create a new, detached task that produces a value of type `T`.
 @discardableResult
-static func detach<T: Sendable>(
```

Comment:
I'll actually confirm what should be done here... this was in flux still recently

Reply:
Okey this turned out to be tricky, the change is good though -- no Sendable on the `T` BUT we force it on the `Task.Handle<T, ...>`